aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -461,13 +465,15 @@ module aptos_token::token {
 
     public fun merge(dst_token: &mut Token, source_token: Token) {
         assert!(&dst_token.id == &source_token.id, error::invalid_argument(EINVALID_TOKEN_MERGE));
+        // TODO: Why not validate that property_version = 0 here?
```

Comment:
deleted this comment. I think an edge case can happen if one token's amount is 0.

Reply:
Okay makes sense